src/plugins/data/public/antlr/opensearch_ppl/code_completion.ts

```diff
@@ -40,16 +40,50 @@ export const getSuggestions = async ({
       finalSuggestions.push(...fetchFieldSuggestions(indexPattern, (f: any) => `${f} `));
     }
 
+    if (suggestions.suggestValuesForColumn) {
+      // get dataset for connecting to the cluster currently engaged
+      const dataset = services.data.query.queryString.getQuery().dataset;
+
+      // take the column and push in values for that column
+      const res = await fetchColumnValues(
+        [indexPattern.title],
+        suggestions.suggestValuesForColumn,
+        services,
+        dataset
+      );
```

Comment:
how does it handle errors?

Reply:
what it would do here is just default to no suggestions, since at this point if res is a rejection from the api, it will be caught by lines 120-122. since the code completion method is called for any partially formed query, it should be fine to not show any suggestions if fetchColumnValues doesn't return anything.